RecoEgamma/EgammaTools/python/calibratedEgammas_cff.py

```diff
@@ -0,0 +1,65 @@
+import FWCore.ParameterSet.Config as cms
+
+calibratedEgammaSettings = cms.PSet(minEtToCalibrate = cms.double(5.0),
+                                    semiDeterministic = cms.bool(True),
+                                    correctionFile = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2017_17Nov2017_v1_ele_unc"),
+                                    recHitCollectionEB = cms.InputTag('reducedEcalRecHitsEB'),
+                                    recHitCollectionEE = cms.InputTag('reducedEcalRecHitsEE'),
+                                    produceCalibratedObjs = cms.bool(True)
+                                    )
+calibratedEgammaPatSettings = calibratedEgammaSettings.clone(
+    recHitCollectionEB = cms.InputTag('reducedEcalRecHitsEB'),
+    recHitCollectionEE = cms.InputTag('reducedEcalRecHitsEE'),
+    )
+
+ecalTrkCombinationRegression = cms.PSet(
+    ecalTrkRegressionConfig = cms.PSet(
+        rangeMin = cms.double(-1.),
+        rangeMax = cms.double(3.0),
+        lowEtHighEtBoundary = cms.double(50.),
+        forceHighEnergyTrainingIfSaturated = cms.bool(True),
+        ebLowEtForestName = cms.string('electron_eb_ECALTRK_lowpt'),
+        ebHighEtForestName = cms.string('electron_eb_ECALTRK'),
+        eeLowEtForestName = cms.string('electron_ee_ECALTRK_lowpt'),
+        eeHighEtForestName = cms.string('electron_ee_ECALTRK')
+        ),
+    ecalTrkRegressionUncertConfig = cms.PSet(
+        rangeMin = cms.double(0.0002),
+        rangeMax = cms.double(0.5),
+        lowEtHighEtBoundary = cms.double(50.),  
+        forceHighEnergyTrainingIfSaturated = cms.bool(True),
+        ebLowEtForestName = cms.string('electron_eb_ECALTRK_lowpt_var'),
+        ebHighEtForestName = cms.string('electron_eb_ECALTRK_var'),
+        eeLowEtForestName = cms.string('electron_ee_ECALTRK_lowpt_var'),
+        eeHighEtForestName = cms.string('electron_ee_ECALTRK_var')
+        ),
+    maxEcalEnergyForComb=cms.double(200.),
+    minEOverPForComb=cms.double(0.025),
+    maxEPDiffInSigmaForComb=cms.double(15.),
+    maxRelTrkMomErrForComb=cms.double(10.),
+    
+)
+
+calibratedElectrons = cms.EDProducer("CalibratedElectronProducer",
+                                     calibratedEgammaSettings,                                   
```

Comment:
the changes made in fillDescriptions were meant to also be able to use
```python 
calibratedElectrons = calibratedElectronProducer_cfi.calibratedElectronProducer.clone( ...
```
it reduces maintenance needs as well as redundant reimplementation of all config parameters inside the module PSet

Reply:
I guess this clean up can also be done in a follow up, since it's technical, not to lead to actual changes in what's finally configured in a job

Reply:
so personally, I kinda dislike that approach, I'm of the school who likes to see everything in the config.

Reply:
you can still see everything in the config in calibratedElectronProducer_cfi.py
Unless you are saying that you dislike cloning a configured module in general and for all long PSets would want to copy-paste.